python/packages/nisar/workflows/get_product_geometry.py

```diff
@@ -52,6 +52,24 @@ def get_parser():
                         choices=['A', 'B'],
                         help='Frequency band: "A" or "B"')
 
+    parser.add_argument('--spacing-y',
+                        dest='spacing_y',
+                        required=False,
+                        default=None,
+                        type=float,
+                        help='Y spacing, the same unit with the L2 product (only'
+                        ' applicable if the input is a NISAR L2 product). Defaults'
+                        ' to the Y-coordinate spacing of the product.')
```

Comment:
```suggestion
                        help='Y spacing, in the same unit as the L2 product. Can'
                        ' be positive or negative. This argument is only'
                        ' applicable if the input is a NISAR L2 product. Defaults'
                        ' to the Y-coordinate spacing of the product.')
```

Just adding that Y spacing can be positive or negative.

Reply:
Thank you @gshiroma